src/api/axios/axios.service.ts

```diff
@@ -0,0 +1,19 @@
+import axios, {AxiosInstance} from 'axios';
+import {inject, injectable} from 'inversify';
+import 'reflect-metadata';
+import {TYPES} from '@/provider/types';
+import {AxiosFactory} from '@/api/axios/axios.factory';
+
+@injectable()
+export class AxiosService {
+  private hatchout: AxiosInstance;
+  constructor(
```

Comment:
How about rename the `hatchout` to `axiosInstance`?

Reply:
i thought we can use other server like naver api or kakao api etc.
if we name axiosInstance, it will be difficult to name to use other server.
how do u think about it?